lib/components/molecules/Popover/Popover.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+/**
+ *props detailed explanation
+ *className - class for the popover wrapper
+ *children - child nodes of the wrapper
+ *body - The popover content
+ *preferPlace - Sets a preference of where to position the Popover.
+    valid values
+      ->above|right|below|left: prefer an explicit side
+      ->row|column: prefer an orientation
+      ->start|end: prefer an order
+      ->null - default, no preference, automatic resolution
+  *place - ????
+  *onOuterAction  - callback that executes every time user does an action
+  *refreshIntervalMs  - time bet each poll in ms. Way to position target in DOM
+  *enterExitTransitionDurationMs - time in ms that it takes to complete enter and exit animation default 500
+  *tipSize : size of tip pointer, default 7
+  *target  - React element to which popover orient itself
+  *appendTarget - Where popover will mount into. Default is 'document.body'
+*/
+import type { Node } from 'react';
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import Popover from 'react-popover';
+import styles from './Popover.style';
+
+type Props = {
```

Comment:
Looks like you have not being following the generator for building this component, because in case of complex molecule we usually define flow types under types/index